src/Mapper.php

```diff
@@ -84,7 +84,7 @@ protected function doMapping(): object
                     continue;
                 }
 
-                throw new MapperException("Property {$attribute->name} is not nullable");
+                throw new MapperException("Property {$attribute->name} on ".$this->instance::class.' not nullable');
```

Comment:
```suggestion
                throw new MapperException(implode('', [
                    'Property ',
                    $attribute->name,
                    " on ",
                    $this->instance::class,
                    ' not nullable'
                ]));
```

Reply:
Why is this something that I actually saw in production once and when I asked a coworker about it, he actually defended it?

Reply:
> Why is this something I've probably done before

Because you have no soul, heartless code abuser. :(

Reply:
```suggestion
                throw new MapperException("Property {$attribute->name} on {$this->reflection->name} not nullable");
```

Reply:
```suggestion
                throw new MapperException(
                    (function ($template, $variables) {
                        $start = 0;
                        while (($start = strpos($template, "{", $start)) !== false) {
                            $end = strpos($template, "}", $start + 1);
                            if ($end === false) {
                                break;
                            }

                            $variableName = substr($template, $start + 1, $end - $start - 1);
                            if (array_key_exists($variableName, $variables)) {
                                $template = str_replace("{" . $variableName . "}", $variables[$variableName], $template);
                            }

                            $start = $end + 1;
                        }

                        return $template;
                    })('Property {aName} on {cName} not nullable', array(
                        'aName' => $attribute->name,
                        'cName' => $this->instance::class
                    ))
                );
```

Reply:
Okay I can at least say I've never done this before...